spec/dummy/client/app/startup/ClientReduxApp.jsx

```diff
@@ -21,11 +22,20 @@ const ReduxApp = props => {
   const combinedReducer = combineReducers(reducers);
   const store = applyMiddleware(middleware)(createStore)(combinedReducer, props);
 
-  const reactComponent = (
-    <Provider store={store}>
+  let reactComponent;
+  if (Common.react013()) {
```

Comment:
Yep.  I ran into this the first day I changed it over to 0.14.  Someone elses PR against the sample project made the same sort of corrections against that project :)